src/auth/Auth.jsx

```diff
@@ -46,7 +48,16 @@ export class Auth extends Component {
   }
 
   componentDidMount() {
-      this.props.fetchTokenData(this.props.storagePrefix);
+    if (this.props.history && this.props.history.location.search) {
+      const query = parseQuery(this.props.history.location.search);
```

Comment:
I tried, but it looks props.history.location.search is empty. It can't get the token info.

Reply:
Are You tried something like http://localhost:8080/?token=<token> ?

Reply:
one more thing, if you open webui and add query it's doesn't work because this is called only when the component is mounted, so after a refresh, it should works

Reply:
I confirmed it works, thanks!